README.md

```diff
@@ -36,20 +36,135 @@
 
 ## Usage
 
-1. Clone this repository locally
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Import the NixOS and [home-manager](https://github.com/nix-community/home-manager) modules
```

Comment:
```suggestion
1. Import the [NixOS](https://nixos.org) and [home-manager](https://github.com/nix-community/home-manager) modules
```
I think it looks more consistent with both as a link, even though https://nixos.org is linked elsewhere.